nimbleparse/src/main.rs

```diff
@@ -325,7 +353,8 @@ fn main() {
                 eprintln!();
             }
             if let Some(token_spans) = missing_from_parser {
-                let formatter = SpannedDiagnosticFormatter::new(&lex_src, &lex_l_path).unwrap();
+                let formatter =
+                    yacc_diag.get_or_init(|| spanned_formatter(&yacc_src, &yacc_y_path));
```

Comment:
This one is using `yacc_src` where it should be using `lex_src`, definitely needs to change.